app/src/main/java/com/fstyle/androidtrainning/screen/main/mainfragments/online/subfragment/toptrack/TopTrackRecyclerAdapter.java

```diff
@@ -0,0 +1,109 @@
+package com.fstyle.androidtrainning.screen.main.mainfragments.online.subfragment.toptrack;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+import com.bumptech.glide.Glide;
+import com.fstyle.androidtrainning.R;
+import com.fstyle.androidtrainning.model.TopTracks;
```

Comment:
ERROR: Unused import: com.fstyle.androidtrainning.model.TopTracks. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports